tika-parsers/src/main/java/org/apache/tika/parser/tagratio/TextToTagRatio.java

```diff
@@ -0,0 +1,238 @@
+package org.apache.tika.parser.tagratio;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.ByteArrayInputStream;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.StringReader;
+import java.net.MalformedURLException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.HashMap;
+
+import org.apache.tika.exception.TikaException;
+import org.htmlparser.Parser;
+import org.htmlparser.util.NodeList;
+import org.htmlparser.util.ParserException;
+import org.json.JSONArray;
+import org.json.simple.JSONObject;
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+import org.jsoup.nodes.Element;
+import org.xml.sax.SAXException;
+
```

Comment:
@chrismattmann 
I am actually planning to add a class for TextToTagRatio inside TxtParser.java and define all the methods inside the class as static methods. Then I'll overload the parse method of Tika by adding one more parse method with boolean argument in addition to the existing parameters. Will use this boolean value to decide whether to invoke TextToTagRatio Parser or not. Is this approach fine or any further suggestions Professor?